src/schema/me/bidder_position_mutation.js

```diff
@@ -0,0 +1,43 @@
+// @ts-check
+
+import { GraphQLString, GraphQLFloat } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+
+export const BidderPositionMutation = mutationWithClientMutationId({
+  name: "BidderPosition",
+  description:
+    "Creates a bidder position",
+  inputFields: {
+    sale_id: {
+      type: GraphQLString,
+    },
+    artwork_id: {
+      type: GraphQLString,
+    },
+    max_bid_amount_cents: {
+      type: GraphQLFloat,
+    },
+  },
+  outputFields: {
+    sale_id: {
+      type: GraphQLString,
+      resolve: (position) => position.bidder.sale.id,
```

Comment:
I was a bit confused about what/how this mutation should return. 🤔

Reply:
It's up to you! Whatever the client needs (and makes general sense) is cool to put here.

You may want to resolve with the result of the gravity POST, so just `resolve: position => position` (and the type what be correspondingly be `BidderPosition.type`).